### ISSUE TITLE
Allow override of hidden cvars

### ISSUE THREAD

**Issue by CitadelCore** [CitadelCore/Portal-2-Community-Edition#6]
Allow development/hidden cvars to be force enabled with a launch parameter.

**Comment by Luke18033** [comment]
There's already [a plugin](http://www.mediafire.com/file/j9jcb53m91s4ycj/csgo_helper.zip) that lets you do this, I guess it would be nice to have as a default option though.

**Comment by CitadelCore** [comment]
Yep, I know about the plugin, just planning to add it in engine code with a launch option, for example `-forceenablecvars`

**Comment by Luke18033** [comment]
Is this still planned? You made it sound like this wasn't going to be an option on Discord earlier.
